problems/numberOfDigits.js

```diff
@@ -7,8 +7,9 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let arr = n.toString().split('').map(Number)
```

Comment:
Theres no need for the map method in this line, the map isn't really doing anything